MANIFEST.in

```diff
@@ -6,6 +6,7 @@ prune examples/*/*/htmlcov
 prune examples/adhoc-layout/*.egg-info
 prune examples/src-layout/src/*.egg-info
 
+graft .github/workflows
```

Comment:
is there a reason for publishing the github workflow to pypi? they both use `actions/checkout@v2` which won't work without a live git repo and can't work from the sdist

Reply:
Yes, the reason is `.travis.yml .appveyor.yml` were/is included below, so I was aiming for parity.

Can remove if you like?

Reply:
I don't know the best option here is.

I think using the "wheel_from_sdist", and then using that wheel in tox is the state of the art for ensuring your MANIFEST is correct https://github.com/pypa/build/issues/257

Reply:
No strong opinion here, although I use setuptools-scm elsewhere and it makes things easy. Something for a followup issue?

Reply:
I think it's fair to include it in the sdist, considering travis stuff was previously included, and it would be useful to have some evidence in the sdist of how that particular release was supposed to be tested.